src/steps/writing/creation/dotVSCode.ts

```diff
@@ -50,5 +60,17 @@ export async function createDotVSCode(options: Options) {
 			"eslint.rules.customizations": [{ rule: "*", severity: "warn" }],
 			"typescript.tsdk": "node_modules/typescript/lib",
 		}),
+		"tasks.json": await formatJson({
+			tasks: [
+				{
+					detail: "Build the project",
+					label: "build",
+					problemMatcher: [],
```

Comment:
[Question] I'm reading the docs on https://code.visualstudio.com/docs/editor/tasks and not really getting what `problemMatcher` does. But if it's blank, can it reasonably be removed?

```suggestion
```

Reply:
doesn't seem to be necessary - have removed